playground/backend/internal/environment/environment_service.go

```diff
@@ -92,3 +94,40 @@ func getSdkEnvsFromOsEnvs() *BeamEnvs {
 
 	return NewBeamEnvs(sdk)
 }
+
+func readConfig(apacheBeamSdk pb.Sdk) *CmdConfig {
+	cmdConfig := getCmdConfig(apacheBeamSdk)
+	switch apacheBeamSdk {
+	case pb.Sdk_SDK_JAVA:
+		cmdConfig.CompileArgs = append(cmdConfig.CompileArgs,
+			getEnv("BEAM_SDK", "/opt/apache/beam/jars/beam-sdks-java-harness.jar"))
+		jars := strings.Join([]string{
+			getEnv("BEAM_SDK", "/opt/apache/beam/jars/beam-sdks-java-harness.jar"),
+			getEnv("BEAM_RUNNER", "/opt/apache/beam/jars/beam-runners-direct.jar"),
+			getEnv("JDK", "/opt/apache/beam/jars/slf4j-jdk14.jar"),
+		}, ":")
+		cmdConfig.RunArgs[1] += jars
+	case pb.Sdk_SDK_GO:
+	case pb.Sdk_SDK_PYTHON:
+	case pb.Sdk_SDK_SCIO:
+	}
+
+	return &cmdConfig
+}
+
+func getCmdConfig(apacheBeamSdk pb.Sdk) CmdConfig {
+	CmdConfigPath := getEnv("CONFIG_FOLDER", "./") + apacheBeamSdk.String() + ".json"
+	cmdConfig := CmdConfig{}
+	err := gonfig.GetConf(CmdConfigPath, &cmdConfig)
+	if err != nil {
+		panic(err)
+	}
+	return cmdConfig
+}
+
+func getEnv(key, fallback string) string {
```

Comment:
don't fully understand why we need this method, however if it reasonable - let's use this to read all envs